modules/ROOT/pages/_partials/commons/common-prebuilt-database.adoc

```diff
@@ -10,24 +10,21 @@
 == Overview
 
 
-{cbl-te}_'s_ support for pre-built databases means you can pre-load your app with data instead of syncing it down from {sgw-te} during startup.
-This might benefit, for example, a mobile application developer striving to minimize consumer wait time (arising from data setup) on initial install and launch of the application.
+{cbl-te} supports  pre-built databases. You can pre-load your app with data instead of syncing it from {sgw-te} during startup to minimize consumer wait time (arising from data setup) on initial install and launch of the application. 
```

Comment:
New line for new sentence.

Reply:
Not urgent for now, but in near future we could look at replacing `{cbl-te}` and friends either with a more readable label or, even 😱 typing the whole term out...